### DISCUSSION TITLE
Multi-platform build from oci-oracle-free with QEMU fails

### DISCUSSION THREAD

**Discussion by catostrophe** [gvenzl/oci-oracle-free#74] [Q&A] [answered]
I'm trying to build a custom image based on oci-oracle-free for different platforms.

It works fine when I run build on both target platforms separately (if I run it either on amd64 or arm64 hardware).

But when I try to make a multi-platform build on CI on an amd64 runner:
```shell
docker run --privileged --rm tonistiigi/binfmt --install all
docker buildx create --name multi-platform-builder
docker buildx use multi-platform-builder
docker buildx inspect --bootstrap
docker buildx build \
        --platform=linux/arm64 \
        --progress=plain \
        --secret id=ORACLE_PASSWORD,env=ORACLE_PASSWORD \
        .
```

I get this:
```
ORA-00443: background process "PMON" did not start
```
It is well known to me that the image only works on a different platform via Colima. 

@gvenzl So how do you build the images for multiple platforms? Do you have separate runners and build images separately and manage manifest manually? Or is there a way to build it on one machine?

**Comment by gvenzl** [comment, marked as answer]
Hey @catostrophe,

Thanks for using these images!

That's exactly how I build them.
I run separate builds on an x86 and an arm64 machine, upload them to the container registry, and then create and push a multi-arch manifest referencing the newly built images.

**Reply by catostrophe** [reply]
Thank you. Your project is a game changer in my team. From local dev instances to one-off containers for CI to create-with-a-push-button persistent instances in the cloud for testing and staging.

**Reply by gvenzl** [reply]
Thank you very much for the kind words, @catostrophe! Let me know if you have anything you want me to call out on https://github.com/gvenzl/oci-oracle-free?tab=readme-ov-file#users-of-these-images and, of course, if I can help with anything else.
